src/main/java/frc/robot/Robot.java

```diff
@@ -258,17 +263,37 @@ public void robotInit() {
         .whileTrue(staticAutoAim());
     controller
         .rightTrigger()
-        .and(() -> currentTarget == Target.SPEAKER)
-        .and(operator.b())
+        .and(() -> currentTarget == Target.FEED)
         .whileTrue(
-            shooter.runStateCmd(
-                AutoAim.FEED_SHOT.getRotation(),
-                AutoAim.FEED_SHOT.getLeftRPS(),
-                AutoAim.FEED_SHOT.getRightRPS()))
+            Commands.parallel(
+                Commands.waitUntil(() -> shooter.isAtGoal())
+                    .andThen(controller.rumbleCmd(1.0, 1.0).withTimeout(0.25))
+                    .asProxy(),
+                shooter.runStateCmd(
+                    AutoAim.FEED_SHOT.getRotation(),
+                    AutoAim.FEED_SHOT.getLeftRPS(),
+                    AutoAim.FEED_SHOT.getRightRPS())))
         .onFalse(
             Commands.parallel(
                     shooter.run(() -> {}), feeder.runVelocityCmd(FeederSubsystem.INDEXING_VELOCITY))
                 .withTimeout(0.5));
+    controller
```

Comment:
Feels a little like this can be [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)'d up since the command appears to be identical aside from the target. Not a strong push, but I worry a bit that you tune one and forget to touch the others. Unless you think they're actually independent?

Reply:
there isnt a great way to abstract the trigger bindings here that doesnt result in more verbosity. it isnt inconceivable that we want these to have more different behaviour later either.